imap_processing/codice/tests/test_bin_data.py

```diff
@@ -0,0 +1,53 @@
+""" This code is used to test the size of a binary file. It is not used in the actual decom.py file.
+    It is used to test the size of the binary file that is used in the decom.py file.
+    """
+
```

Comment:
I think it is out of scope for this PR, but an improvement to be made here is to reformat this module to reflect more of a `pytest` style, so that these tests can be automatically run (along with the other `imap_processing` tests) during the GitHub Actions builds.